website/docs/r/database.html.markdown

```diff
@@ -686,7 +686,16 @@ Review the argument reference that you can specify for your resource.
 - `service` - (Required, Forces new resource, String) The type of Cloud Databases that you want to create. Only the following services are currently accepted: `databases-for-etcd`, `databases-for-postgresql`, `databases-for-redis`, `databases-for-elasticsearch`, `messages-for-rabbitmq`,`databases-for-mongodb`,`databases-for-mysql`, and `databases-for-enterprisedb`.
 - `service_endpoints` - (Required, String) Specify whether you want to enable the public, private, or both service endpoints. Supported values are `public`, `private`, or `public-and-private`.
 - `tags` (Optional, Array of Strings) A list of tags that you want to add to your instance.
-- `version` - (Optional, Forces new resource, String) The version of the database to be provisioned. If omitted, the database is created with the most recent major and minor version.
+- `version` - (Optional, String) The version of the database to be provisioned or upgraded to. If omitted, the database is created with the latest supported major and minor version. This field can be updated to perform an in-place upgrade without forcing the creation of a new resource. The database will be put into READ-ONLY mode during upgrade. It is highly recommended to test before upgrading. To learn more, refer to the version upgrade documentation.
+
+  > ⚠️ **Warning:** Upgrading may require more time than the default timeout.  
+  > A longer timeout value can be set using the timeouts attribute.
+
```

Comment:
Can we provide a link to version upgrade documentation where we say "refer to the version upgrade documentation"

Reply:
The docs for upgrades are at a database level. eg https://test.cloud.ibm.com/docs/databases-for-mongodb?topic=databases-for-mongodb-upgrading&interface=terraform

So as more dbs adopt they might be differences in docs based on db type so I didnt want to add for cli and terraform

Reply:
I could ask docs team to add a high level one that links to all dbs but they are still working on the docs for the upgrade itself so wont have that done in time for release. I can add a ticket though?